css/style.css

```diff
@@ -0,0 +1,84 @@
+/**
+ * Place your custom styles here.
+ */
+
+ /*GLOBAL OVERRIDES*/
+@import url('https://fonts.googleapis.com/css?family=Raleway');
+@import url('https://fonts.googleapis.com/css?family=Oswald');
+p,li,dt,dd,td,th,details,button,form,a{font-family: Raleway;}
```

Comment:
Fallbacks are in the parent Bootstrap - don't want to declare the same families twice so only use the variable fonts here.